src/main/java/org/terasology/blockNetwork/NetworkNode.java

```diff
@@ -19,10 +19,29 @@
 import org.terasology.math.SideBitFlag;
 import org.terasology.math.geom.Vector3i;
 
+/**
+ * Represents a single node in a block network.
+ * Any instance of this class or its children can be used as a node in {@link EfficientNetwork}
+ */
 public class NetworkNode {
+    /**
+     * The location of the Node
+     */
```

Comment:
This is not javadoc, this is a simple comment. Javadoc is only for methods (see https://en.wikipedia.org/wiki/Javadoc#Structure_of_a_Javadoc_comment). For single line comments such as this, it is preferred that you use //.

Reply:
Also, since this is a comment, the programmer can see the comment AND the code, unlike javadocs (which are mostly seen as only javadocs by users of the API). So, it will probably not surprise most programmers that a variable named `location` hold the location of the node. I would recommend you remove this comment, unless you have something informative to add here.